test/SAPHanaSR-replay-archive

```diff
@@ -135,20 +174,21 @@ for sid in $sids; do
        cib="${cibBZ2%.bz2}";
        shortCIB="${cib##*/}"
        echo "cib: $cib";
-       bunzip2 -c "$cibBZ2" > "${archive}-tmp/tmp/$shortCIB";
+       cibFile="$pengine_tmp"/"$shortCIB"
+       less -c "$cibBZ2" > "$cibFile";
        case "$format" in
            tables | script )
-               /usr/sbin/SAPHanaSR-showAttr --cib="${archive}-tmp/tmp/$shortCIB" --sid="$sid" --format="$format";
+               /usr/sbin/SAPHanaSR-showAttr --cib="$cibFile" --sid="$sid" --format="$format";
                    ;;
            book )
                if [ -n "$outDir" ]; then
                    outOption="--out=$outDir/sapMonitor.$outNr.html"
                    (( outNr ++ ))
                fi
-               /usr/sbin/SAPHanaSR-monitor --cib="${archive}-tmp/tmp/$shortCIB" --sid="$sid" --format="html2" "$outOption";
+               /us/sbin/SAPHanaSR-monitor --cib="$cibFile" --sid="$sid" --format="html2" "$outOption";
```

Comment:
Typo. /us/ instead of /usr/

Reply:
@angelabriel: Many thanks for beeing checking that carefully.